diagnostic.md

```diff
@@ -185,4 +221,6 @@ with the class `.stretch` be at a _screen width_ of 550px?
 
 <!-- your answer starts here -->
 
+25px
```

Comment:
at 550px, the `@media(min-width= 400px)` would take over and `.stretch` is 25%. So 25% * 550px = 137.5px.